### PR TITLE
Fix old update info

### PR DESCRIPTION
**Pull request by linknum23**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,8 +107,8 @@ To update you AmpliPi to the latest version:
 1. Click the **Update** button
 
 If you don't see an update button, you have an older version of AmpliPi. It will take a couple more steps to update this time around.
-1. Download this [update](https://github.com/micro-nova/AmpliPi/releases/download/0.1.7/amplipi-update-0.1.7.tar.gz).
-1. Click the browse button to select the downloaded amplipi-update-0.1.7.tar.gz file.
+1. Download this [update](https://github.com/micro-nova/AmpliPi/archive/refs/tags/0.1.9-uv-hotfix.tar.gz).
```
Comment (linknum23) [inline README.md:110]:
```suggestion
1. From a laptop or desktop computer download this [update](https://github.com/micro-nova/AmpliPi/archive/refs/tags/0.1.9-uv-hotfix.tar.gz).
```

File: README.md [hunk 2]
```diff
@@ -107,8 +107,8 @@ To update you AmpliPi to the latest version:
...
+1. Click the browse button to select the downloaded AmpliPi-0.1.9-uv-hotfix.tar.gz file.
 1. Click upload software to start the update, when it is finished it will navigate to the updated web app.
 1. Please be patient, updates can take 10-15 minutes and progress info will be reported slowly.
```
Comment (linknum23) [inline README.md:113]:
```suggestion
1. Please be patient, updates can take 20-30 minutes and progress info will be reported slowly.
```

**Comment by linknum23** [comment]
The link on the README is out of date since we removed the 0.1.7 release. Lets fix this and add more specific instructions for those that are still using the original release that came with early AmpliPis.

**Comment by linknum23** [comment]
To get ci to pass we need httpx dep

**Comment by linknum23** [comment]
@klay2000 can you check this out? It should solve any CI problems as well.

**Review by klay2000 — APPROVED** [review]

**Pull request merged**